core/trino-main/src/main/java/io/trino/operator/output/BlockBuilderPositionsAppender.java

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
add JMH benchmark results as commit message description

Reply:
The full benchmarks are really long (in terms of the number of lines) and this makes the IntelliJ commit message dialog to hang when using auto-completion.
How about I post the full jmh in the PR description and only subset here?